src/index.js

```diff
@@ -239,7 +239,9 @@ var enqueuePostPromiseJob =
         process.nextTick(fn);
       });
     } :
-    setImmediate || setTimeout;
+    typeof setImmediate === 'function' ?
+      setImmediate :
+      fn => { setTimeout(fn, 0); };
```

Comment:
At first I wrote as following but it caused error on test.

```javascript
    typeof setImmediate === 'function' ? setImmediate : setTimeout;
```

```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ src/index.js:401:12

Cannot return enqueuePostPromiseJob because TimeoutID [1] is incompatible with
undefined [2] in the return value.

     src/index.js
 [2] 398│ ): (() => void) => void {
     399│   var batchScheduleFn = options && options.batchScheduleFn;
     400│   if (batchScheduleFn === undefined) {
     401│     return enqueuePostPromiseJob;
     402│   }
     403│   if (typeof batchScheduleFn !== 'function') {
     404│     throw new TypeError(

     /tmp/flow/flowlib_20a9f9de/core.js
 [1] 819│ ): TimeoutID;
```